.eslintrc.js

```diff
@@ -72,7 +72,7 @@ module.exports = {
     'prefer-rest-params': 'error',
     'prefer-spread': 'error',
     'prefer-template': 'error',
-    quotes: ['error', 'single', { avoidEscape: true, allowTemplateLiterals: false }], // Disallow unnecessary template literals.
+    quotes: ['error', 'single', { avoidEscape: true, allowTemplateLiterals: true }],
```

Comment:
I'm not sure why this was set to `false`, as this seemed to cause issues with inline snapshots 🤷